spp_change_request/models/change_request.py

```diff
@@ -239,6 +266,17 @@ def _check_applicant_phone(self):
 
     @api.onchange("id_document_details")
     def _onchange_scan_id_document_details(self):
+        """
+        This method is called whenever there is a change in the value of id_document_details field.
+
+        Updates other fields based on the field value of id_document_details
+
+        id_document_details must be a JSON Serializable
+
+        :return:
+
+        :raise UserError: Exception raised when something is not valid.
+        """
```

Comment:
Probably need to explain that this si used in conjunction with an ID document scanner